src/app/core/audit/audit.controller.ts

```diff
@@ -1,20 +1,18 @@
-'use strict';
+import _ from 'lodash';
 
-const _ = require('lodash'),
-	deps = require('../../../dependencies'),
-	dbs = deps.dbs,
-	util = deps.utilService,
-	Audit = dbs.admin.model('Audit');
+import { dbs, utilService as util } from '../../../dependencies';
+
+const Audit = dbs.admin.model('Audit');
```

Comment:
Type should be applied here:

```ts
const Audit: AuditModel = dbs.admin.model('Audit');
```

Doing so will ensure proper type checking and improve editor autocompletion.  Adding type as above actually causes errors when running `npm run tsc:check` due to missing plugin types on the model/schema.